webdriver/tests/perform_actions/wheel.py

```diff
@@ -0,0 +1,67 @@
+import pytest
+
+from webdriver.error import NoSuchWindowException
+
+from tests.perform_actions.support.refine import filter_dict, get_events
+from tests.support.asserts import assert_move_to_coordinates
+
+
+def test_null_response_value(session, wheel_chain):
+    value = wheel_chain.scroll(0, 0, 0, 10).perform()
+    assert value is None
+
+
+def test_no_browsing_context(session, closed_window, wheel_chain):
+    with pytest.raises(NoSuchWindowException):
+        wheel_chain.scroll(0, 0, 0, 10).perform()
+
+
+def test_wheel_scroll(session, test_actions_scroll_page, wheel_chain):
+    session.execute_script("document.scrollingElement.scrollTop = 0")
+
+    outer = session.find.css("#outer", all=False)
+    wheel_chain.scroll(0, 0, 5, 10, origin=outer).perform()
+    events = get_events(session)
+    assert len(events) > 0
+    event_types = [e["type"] for e in events]
```

Comment:
Why don't we just filter for `type` here? This would prevent us from having the additional for loop. This also applies to the other tests in this file.

Reply:
Done.